flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FuelPredPage.js

```diff
@@ -148,9 +148,13 @@ class CDUFuelPredPage {
                 };
                 if (mcdu.altDestination) {
                     altIdentCell = mcdu.altDestination.ident;
-                    altEFOBCell = (mcdu.getAltEFOB(true) * mcdu._conversionWeight).toFixed(1);
-                    altTimeCellColor = "[color]green";
+                    const altEFOBCellVal = (mcdu.getAltEFOB(true) * mcdu._conversionWeight).toFixed(1);
```

Comment:
what's the purpose of this const?

Reply:
its the current alt dest efob value or what do you mean ?

Reply:
resolved as we talked.